build/configs/common_download.sh

```diff
@@ -231,6 +231,7 @@ function get_partition_sizes()
 
 download_specific_partition()
 {
+	FLAG=$1	
```

Comment:
I don't get what the `FLAG` does. Could you explain? Flag for??

Reply:
I change it and now doing it in different way